src/main/java/me/dinowernli/grpc/prometheus/Common.java

```diff
@@ -0,0 +1,60 @@
+package me.dinowernli.grpc.prometheus;
+
+import io.grpc.Metadata;
+import io.grpc.Metadata.Key;
+import io.prometheus.client.SimpleCollector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Encapsulates some common utilities used in both client and server metrics.
+ */
+class Common {
+    /**
+     * Merges two string lists into an array, maintaining order of first list then second list.
+     */
+    static String[] asArray(List<String> firstList, List<String> secondList) {
+        List<String> list = new ArrayList<>(firstList);
+        list.addAll(secondList);
+        return list.toArray(new String[0]);
+    }
+
+    /**
+     * Converts a list of strings to a list of grpc metadata keys.
+     */
+    static List<Metadata.Key<String>> metadataKeys(List<String> headerNames) {
+        List<Metadata.Key<String>> keys = new ArrayList<>();
+        for (String name : headerNames) {
+            keys.add(Metadata.Key.of(name, Metadata.ASCII_STRING_MARSHALLER));
```

Comment:
nit (here and throughout): can we stick to the 2-space indent present in the rest of the project?